includes/models/model.llms.quiz.php

```diff
@@ -51,7 +51,9 @@ class LLMS_Quiz extends LLMS_Post_Model {
 	 *
 	 * Array key is the meta_key and array values is property's type.
 	 *
-	 * @var string[]
+	 * @since 3.3.0
```

Comment:
where did you take this from ?

Reply:
perhaps a wrong copy/paste. Updated now.